### DISCUSSION TITLE
ssh

### DISCUSSION THREAD

**Discussion by MACCAMENZIES** [xblax/flashforge_ad5m_klipper_mod#226] [Q&A]
Trying to ssh into the klipper mod on the machine to adjust some code.  Its asking for a ssh password however. Does anyone know how to enter it?

**Comment by KaruroChori** [comment]
From the readme of this repo

> SSH root access. Login: root, Password: klipper

**Reply by MACCAMENZIES** [reply]
Many thanks for that
